### PR TITLE
Added blog post about NVD3

### PR DESCRIPTION
**Pull request by argelius**
@frankdiox @andipavllo 
Can you take a look at this post?

### CODE REVIEW

File: blog/posts/2015-07-30.markdown [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+        // This function is called when the data is changed.
+        var update = function() {
+          d3.select(svg[0])
+            .datum(scope.data)
+            .call(chart);
+        };
+
+        // Render the chart every time the data changes.
+        // I serializie the data in order to easily check if the data has changed.
```
Comment (andipavllo) [inline blog/posts/2015-07-30.markdown]: Serialize

File: blog/posts/2015-07-30.markdown [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+<!-- more -->
+
+Some other popular JavaScript libraries are:
+
+* [D3.js](http://d3js.org/) - D3 stands for _Data-Driven Documents_. It's a very powerful library for data visualization that uses SVG. It's quite low-level so even making a simple chart can be a quite complex task, but the end result is stunning.
+* [Chart.js](http://www.chartjs.org/) - Simpler to use than D3.js but still very customizable. Instead of SVG it's using canvas, so it's a very good option if you need to support older browsers or WebViews.
+* [n3-charts](http://n3-charts.github.io/line-chart/#/) - This library is built on top of AngularJS and D3.js. It makes displaying charts in an AngularJS a breeze.
+
+We've recently been discussing if we should support a charting library in Onsen UI, and if so which one we should choose. One of the best candidates is NVD3 since it provides a good combination of ease-of-use and the powerful data visualization of D3.js. It's also very simple to integrate in AngularJS apps, as we will see in this article.
```
Comment (andipavllo) [inline blog/posts/2015-07-30.markdown]: no comma needed after Onsen UI

File: blog/posts/2015-07-30.markdown [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+```
+
+Saving and reloading the browser will show something like this:
+
+<p data-height="266" data-theme-id="13819" data-slug-hash="JdeOLd" data-default-tab="result" data-user="argelius" class='codepen'>See the Pen <a href='http://codepen.io/argelius/pen/JdeOLd/'>JdeOLd</a> by Andreas Argelius (<a href='http://codepen.io/argelius'>@argelius</a>) on <a href='http://codepen.io'>CodePen</a>.</p>
+<script async src="//assets.codepen.io/assets/embed/ei.js"></script>
+
+### Wrapping a NVD3 chart with an AngularJS directive
+
+One of the strength of AngularJS is data-binding, so let's make an AngularJS directive where we can bind the chart data to an HTML attribute. In this section we will create a `<line-chart>` component that can be used in the following way:
```
Comment (andipavllo) [inline blog/posts/2015-07-30.markdown]: Better to put a comma after "in this section"

File: blog/posts/2015-07-30.markdown [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+        y: 16
+      }, {
+        x: 5,
+        y: 25
+      }]
+    }];
+  }]);
+```
+
+At this point we have not written any code for the directive so only the input elements will work. Implementing the directive is pretty straight forward and we'll borrow some code from the previous pure JavaScript example. I've used two services (`d3` and `nv`) that just return the respective objects.
```
Comment (andipavllo) [inline blog/posts/2015-07-30.markdown]: Better to put comma after "at this point"

File: blog/posts/2015-07-30.markdown [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+You can play around with the chart below. Because of the data-binding the chart will be re-rendered every time the data changes.
+
+<p data-height="580" data-theme-id="13819" data-slug-hash="VLVyjM" data-default-tab="result" data-user="argelius" class='codepen'>See the Pen <a href='http://codepen.io/argelius/pen/VLVyjM/'>NVD3 Line Chart Sample</a> by Andreas Argelius (<a href='http://codepen.io/argelius'>@argelius</a>) on <a href='http://codepen.io'>CodePen</a>.</p>
+<script async src="//assets.codepen.io/assets/embed/ei.js"></script>
+
+A directive similar to this one was used in the example at the start of the article. The code can be found on my [GitHub page](https://github.com/argelius/onsen-world-pop).
+
+### Conclusion
+
+If you have any questions about the content of the article please write a comment below. In the case you found some mistake or you would like to improve the content, feel free to make a pull request to the [website repository](https://github.com/OnsenUI/onsen.io/tree/master/blog/posts). We appreciate all kinds of feedback!
```
Comment (andipavllo) [inline blog/posts/2015-07-30.markdown]: "In case you find some mistakes,...." should be more correct

File: blog/posts/2015-07-30.markdown (hunk 3 above)
Comment (argelius) [inline blog/posts/2015-07-30.markdown]: I don't really like commas :)

**Pull request merged**